src/wp-includes/html-api/class-wp-html-span.php

```diff
@@ -31,23 +32,23 @@ class WP_HTML_Span {
 	public $start;
 
 	/**
-	 * Byte offset into document where span ends.
+	 * Byte length of span.
 	 *
-	 * @since 6.2.0
+	 * @since {WP_VERSION}
```

Comment:
Let's not remove 6.2.0

Reply:
I think this is actually a defect of the diff view. I'm not actually removing the old comment, but removing the old property and its attached docblock. this is the new `@since` tag for the new `$length` property, and the `@since` tag for the now-removed `$end` property is gone.

at first I added it back in but I found that more confusing, and the class itself has a `@since` tag noting the removal of the attribute. thoughts?